utils/clickhouse.cc

```diff
@@ -0,0 +1,87 @@
+/**
+ *		Tempesta FW
+ *
+ * Copyright (C) 2024 Tempesta Technologies, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License,
+ * or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program; if not, write to the Free Software Foundation, Inc., 59
+ * Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+#include <iostream>
+
+#include <clickhouse/base/socket.h>
+#include <clickhouse/client.h>
+
+#include "clickhouse.hh"
+
+constexpr std::chrono::milliseconds max_wait(100);
+constexpr size_t max_events = 1000;
+constexpr char table_creation_querry[] = "CREATE TABLE IF NOT EXISTS "
+	"access_log (timestamp DateTime64, address IPv6, method UInt8, "
```

Comment:
I have a question why method is not string? It seems that number is internal for Tempesta and not useful for administrator

Reply:
As we discussed, integer is more lightweight. I described methods/numbers compliance in wiki.